go-controller/pkg/util/net_linux.go

```diff
@@ -170,25 +170,43 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string) {
+func DeleteConntrack(ip string, port int32, protocol kapi.Protocol) error {
 	ipAddress := net.ParseIP(ip)
 	if ipAddress == nil {
-		klog.Errorf("Value \"%s\" passed to DeleteConntrack is not an IP address", ipAddress)
-		return
+		return fmt.Errorf("value %q passed to DeleteConntrack is not an IP address", ipAddress)
 	}
 
 	filter := &netlink.ConntrackFilter{}
+	if protocol == kapi.ProtocolUDP {
+		// 17 = UDP protocol
+		err := filter.AddProtocol(17)
```

Comment:
@JacobTanenbaum `if err := filter.AddProtocol(17); err != nil {` and alos a couple places below...

Reply:
done